test/metabase/query_processor_test/expressions_test.clj

```diff
@@ -677,8 +710,9 @@
                                 :expressions  {"One" [:value 1.0 {:base_type :type/Float}]
                                                "Two" [:value 2 {:base_type :type/Integer}]
                                                "Bob" [:value "Bob's Burgers" {:base_type :type/Text}]}
-                                :filters      [[:= 2.0 [:* [:expression "Two"] [:expression "One"]]]]}
```

Comment:
Mixed up the filter syntax between mlv2 and legacy 😬 . These filters all evaluated to `true` and hence didn't affect the results, but they are now actually getting compiled rather than dropped.